src/components/pages/account/InvestmentInfo.tsx

```diff
@@ -21,20 +19,12 @@ export type InvestmentInfoProps = {
 export default function InvestmentInfo({
   account,
 }: InvestmentInfoProps): JSX.Element {
-  const { data: interval } = useInterval();
   const { data: investment } = useInvestment(account.guid);
-  let { data: prices } = usePrices({ from: account.commodity });
 
-  if (!investment || !prices) {
+  if (!investment) {
     return <Loading />;
   }
 
-  prices = prices || [];
-  const latestPrice = prices.getInvestmentPrice(
-    investment.account.commodity.mnemonic,
-    interval.end as DateTime,
-  );
```

Comment:
We already have the current price inside the InvestmentAccount thanks to this change so removing redundant data and using the info inside `quoteInfo`